tap_yotpo/http.py

```diff
@@ -154,6 +149,11 @@ def authenticate(self):
         data = response.json()
         self._token = data['access_token']
 
+    @backoff.on_exception(backoff.expo,
```

Comment:
Moved this to the GET method as the request is created here. After authentication new request wasn't created earlier. Hence moved Backoff to the GET method and now it's working as expected.